x-pack/plugin/search-business-rules/src/yamlRestTest/resources/rest-api-spec/test/search-business-rules/10_pinned_retriever.yml

```diff
@@ -0,0 +1,406 @@
+setup:
+  - do:
+      indices.create:
+        index: test-index1
+        body:
+          settings:
```

Comment:
Shard/replicas shouldn't matter here, it's better to take this off to allow for random test runners.